app/static/src/app/components/VerticalCollapse.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div class="collapse-title p-2">
```

Comment:
Could move `@click="toggleCollapse" `to here to make the whole bar responsive rather than just the chevron at the end

Reply:
Good idea.